expo-maps/src/Map.tsx

```diff
@@ -64,6 +74,26 @@ export class ExpoMap extends React.Component<ExpoMapViewProps> {
     geojsons: [],
   };
   _ismounted = false;
+  mapView = React.createRef<React.Component<any>>();
```

Comment:
I think we can actually improve typing here, instead `React.Component<any>` we can use `ExpoMap`

Reply:
Yes, good idea!